ui/client/common/ClipboardUtils.js

```diff
@@ -0,0 +1,18 @@
+const readText = event => (event.clipboardData || window.clipboardData).getData('text')
+
+const writeText = (text, elementId) => {
```

Comment:
maybe some comment why this is needed and reference where does it come from?

Reply:
Done.